docs/source/teams/installation.rst

```diff
@@ -177,6 +177,14 @@ If your datasets include cloud-backed
 cross-origin resource sharing (CORS) for your cloud buckets. Details are
 provided below for each cloud platform.
 
+Browser Caching
+-----------------
+
+If your datasets include cloud-backed media, we recommend configuring your data
+sources to allow for built in browser caching. This will cache signed URL responses
```

Comment:
+1 - for the other comments I can include the default but I feel like in general we should just tell them to do it instead of hoping that the browser or storage solution handles it properly. Like Stuart mentioned on the ticket some of these "defaults" change based on how you've setup your storage permissions so it's possible our default is different from theirs just based on how we have configured our data to begin with. I'm afraid to say (as an example) "S3 doesn't provide cache headers by default" but if their bucket has X configuration it starts some default value.

Reply:
Valid. I'm not aware of anything you can configure bucket-level that affects that particular metadata in either S3 or GCP - which is why I insisted on including it - but I defer to you - what you're saying makes sense.

Reply:
I was looking around online and seems like GCP is the only one providing a default value (added these in a commit) but probably best to also strongly recommend them to configure it for themselves instead of taking the gamble.